OIIInspector/OIIIClient.py

```diff
@@ -1,9 +1,38 @@
-import json
+from LocalExecutor import LocalExecutor
+from GrpCurlAddressImporter import GrpCurlAddressImporter
+import ast
 
 
 class OIIIClient:
 
-    def get_index_image_package_list(self, image_address, address_type):
+    def __init__(self):
+        self.grp_curl_position = GrpCurlAddressImporter().grp_curl_address
+        self.executor = LocalExecutor()
 
-        print(image_address, address_type)
-        return json.loads("{}")
+    def get_index_image_package_list(self, image_address):
+        command_to_call = '{grp_curl_position} {call_type} {image_address} {call_argument}'.format(
+            grp_curl_position=self.grp_curl_position,
+            call_type="-plaintext",
```

Comment:
Do we expect that this might change? It's okay to hard-code it into the command template if we expect it to be the same everywhere